cmake/adjust_global_compile_flags.cmake

```diff
@@ -12,15 +12,6 @@ if (ANDROID)
   set(CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} -Wl,-z,max-page-size=16384")
 endif()
 
-# Suggested by https://gitlab.kitware.com/cmake/cmake/-/issues/20132
```

Comment:
could you elaborate on why this is no longer needed?
the linked CMake issue is still open.

Reply:
Should have been resolved in this commit: https://gitlab.kitware.com/cmake/cmake/-/commit/2785364b7ba32de1f718e9e5fd049a039414a669 ?

Reply:
How can we verify it ?

Reply:
test packaging pipeline build passed